src/containers/Page/components/Footer.tsx

```diff
@@ -301,7 +299,7 @@ export const Footer = () => {
   return (
     <FooterBlock>
       {!!config.zendeskWidgetKey && (
-        <StyledZendesk id="zendesk" locale={zendeskLanguage} widgetKey={config.zendeskWidgetKey}>
```

Comment:
I følge https://support.zendesk.com/hc/en-us/articles/4408821324826-Zendesk-language-support-by-product er det kun "no" som er godkjent for zendesk.

Reply:
Fant denne https://docs.google.com/spreadsheets/d/1-h_Fc3NwfDgjNbfdNAcj4llExNvkhJdWqXQnBfkFPPw/edit?gid=1577384119#gid=1577384119
Ser ut som om nn og nb tolkes som varianter av no.